<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTranspose</class>
 <widget class="QDialog" name="DialogTranspose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chord transposition</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelChroma">
         <property name="text">
          <string>Chroma to transpose</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialChroma">
         <property name="minimum">
          <number>-12</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxChorma">
         <property name="minimum">
          <number>-12</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelRange">
         <property name="text">
          <string>On</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxRange">
         <item>
          <property name="text">
           <string>Only current chord</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Only current line</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Only current song</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WholeDocument</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonApply">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
